client/web/src/repo/commits/GitCommitNode.tsx

```diff
@@ -7,7 +7,7 @@ import { capitalize } from 'lodash'
 
 import { Timestamp } from '@sourcegraph/branded/src/components/Timestamp'
 import { pluralize } from '@sourcegraph/common'
-import { Button, ButtonGroup, Link, Icon, Code, screenReaderAnnounce, Tooltip } from '@sourcegraph/wildcard'
+import { Button, ButtonGroup, Link, Icon, Code, screenReaderAnnounce, Tooltip, ErrorAlert } from '@sourcegraph/wildcard'
```

Comment:
I guess the linter will fail otherwise
```suggestion
import { Button, ButtonGroup, ErrorAlert, Link, Icon, Code, screenReaderAnnounce, Tooltip } from '@sourcegraph/wildcard'
```

Reply:
Hmm, vscode added that and the linter didn't fail. 🤔
Is that enforced usually?

Reply:
Maybe it's not enforced and the linter won't fail, we just usually them in an alphabetic order.

Ah, the linter would've failed if the import statements themselves (e.g. import lines) were out of alphabetic order.

Sorry for confusion!

Reply:
No worries - I prefer them in the alphabetical order too. Thanks for the call out!